src/PresentationalComponents/Cards/Pathways.js

```diff
@@ -69,7 +69,9 @@ const PathwayCard = (props) => {
       </CardBody>
       <CardBody className="body pf-u-font-size-sm">{description}</CardBody>
       <CardBody className="body pf-u-font-size-sm">
-        {has_incident && <RuleLabels rule={{ tags: 'incident' }} isCompact />}{' '}
+        {has_incident && (
+          <RuleLabels rule={{ tags: 'incident' }} isCompact noMargin={true} />
```

Comment:
It seems to me that we can remove ={true} and just pass noMargin.
```suggestion
          <RuleLabels rule={{ tags: 'incident' }} isCompact noMargin />
```